### ISSUE TITLE
Add originator attribute to AbstractArtifact

### ISSUE THREAD

**Issue by goneall** [cdfoundation/sig-security-sbom#12]
Originator is an optional attribute of Package in SPDX.

It is an important field for several use cases.  Although some of the more detailed (and complex) types and classes in SBOM may provide similar information, having a simple text field would be quite useful and allow for compatibility.

**Comment by CASTResearchLabs** [comment]
the current proposition already has a "supplier" attribute and would handle any additional information as annotation
this was an attempt to keep the model lean

**Comment by goneall** [comment]
I would disagree with the annotation approach, but perhaps replacing it with some of the other more detailed pedigree fields would be appropriate.
